src/status_im/node/core.cljs

```diff
@@ -28,7 +28,7 @@
     (assoc config
            :MaxPeers        20
            :MaxPendingPeers 20
-           :LogLevel        log-level
+           :LogLevel        "DEBUG"
```

Comment:
can it be a keyword? 🤔

Reply:
oh sorry, still in draft, it will be reverted as it was, it was just for debugging :)